db/seeds.rb

```diff
@@ -0,0 +1,15 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+
+Greetings.create([
```

Comment:
- [ ] Also, please make sure to fix the typo in your seed data, it should be `"Greeting"` as the model name is in Singular. 🤓 🔧